iteration/iteration_test.go

```diff
@@ -307,7 +301,7 @@ func (s *TestIterationRepository) TestCreateIterationSameNameFailsWithinSpace()
 	i := iteration.Iteration{
 		Name:       name,
 		SpaceID:    space1.ID,
-		UserActive: &userActive,
+		UserActive: false,
```

Comment:
Remove this line, since `UserActive` will automatically be `false`.

Reply:
Removed it here b92020c